orc8r/cloud/go/tools/swaggergen/generate/generate.go

```diff
@@ -242,6 +288,17 @@ func StripAndWriteSwaggerSpecs(specs map[string]MagmaSwaggerSpec) error {
 	return nil
 }
 
+// RetrieveServiceName parses the targetFilepath for its originating service.
+func RetrieveServiceName(targetFilePath string, rootDir string) (string, error) {
+	absTargetFilepath, err := filepath.Abs(targetFilePath)
+	if err != nil {
+		return "", errors.Wrapf(err, "target filepath %s is invalid", targetFilePath)
```

Comment:
"standalone output filepath %s is invalid"

Reply:
The error is actually coming from reading the target file path and not the output directory. The output path is constructed after.